ButterSample/ButterSample/SampleEndpoints/SampleEndpointA.swift

```diff
@@ -12,11 +12,23 @@ struct SampleEndpointA: Endpoint {
     
     var port: Int? { 8070 }
     
-    var path: String? { "/test/a" }
+    var path: String? { "movies" }
```

Comment:
guessing this should be `/movies`

Reply:
i'll check on this.. i think when you do url appending path components it adds in the first "/" for you. this is hitting the local server ok

Reply:
so it looks like you can do it with or without the leading `/`. so technically it doesn't matter, but it is more readable to have it start with `/` ?

Reply:
AF leaves it out, i'm gonna leave it out as well.